### PR TITLE
remove todo comments

### LINKED ISSUE #1149: Remove todo comments
@inyono @kulla Do you remember what the intention was behind eslint warn about todo comments?

Should we convert them into issue or remove them from the code base? Any opinions?

### PR DESCRIPTION
**Pull request by Entkenntnis**
fixes #1149

### CODE REVIEW

File: src/fetcher/user/request.ts [hunk 1]
```diff
@@ -20,15 +20,15 @@ export async function requestUser(
   })
 
   if (uuid.__typename === 'User') {
-    // TODO: Need to be implemented in the API
+    // stub: Need to be implemented in the API
```
Comment (elbotho) [inline src/fetcher/user/request.ts:23]: what do you mean by stub?

Comment (elbotho) [reply src/fetcher/user/request.ts:23]: but yes, note is enough here. already has an issue:
https://github.com/serlo/frontend/issues/935

**Review by elbotho — APPROVED** [review]
Thank you!
Made a tiny change and added a question, but can be merged.

File: src/fetcher/user/request.ts (hunk 1 above)
Comment (Entkenntnis) [reply src/fetcher/user/request.ts:23]: Stub in the sense of mock / placeholder implementation

**Pull request merged**